pkgs/os-specific/linux/firmware/ipu6-camera-bins/default.nix

```diff
@@ -40,13 +33,13 @@ stdenv.mkDerivation (finalAttrs: {
       include \
       $out/
 
-    install -m 0644 -D ../LICENSE $out/share/doc/LICENSE
+    install -m 0644 -D LICENSE $out/share/doc/LICENSE
 
     runHook postInstall
   '';
 
   postFixup = ''
-    for pcfile in $out/lib/pkgconfig/*.pc; do
+    for pcfile in $out/lib/*/pkgconfig/*.pc; do
       substituteInPlace $pcfile \
```

Comment:
The only replace needed here is `prefix=/usr' "prefix=$out`, everything else is already correct upstream.